clearnode/.gitignore

```diff
@@ -22,8 +22,8 @@
 go.work
 go.work.sum
 
-# env file
-.env
```

Comment:
Why this removal? It seems correct to me to ignore dev's local configuration files such as envrc et similia